meal-planner/src/App.js

```diff
@@ -123,25 +35,26 @@ export default function App() {
                 ) : (
                     ''
                 )}
-                {weekdays.map(({ weekday, id, meal }) => (
-                    <div key={id}>
-                        <Button
-                            day={weekday}
-                            id={weekday}
-                            meal={meal}
-                            onClick={handleClick}
-                        />
-                        {selectedWeekday === weekday && (
-                            <MealSelectMenu
-                                handleMealClick={selectMeal}
-                                weekdayId={id}
-                                handleRecipeClick={handleShowRecipe}
+                {weekdays.length > 0 &&
+                    weekdays.map(({ weekday, id, meal }) => (
+                        <div key={id}>
+                            <Button
+                                day={weekday}
+                                id={weekday}
+                                meal={meal}
+                                onClick={handleClick}
                             />
-                        )}
-                    </div>
-                ))}
+                            {selectedWeekday === weekday && (
+                                <MealSelectMenu
+                                    handleMealClick={selectMeal}
+                                    weekdayId={id}
+                                    handleRecipeClick={handleShowRecipe}
+                                />
+                            )}
+                        </div>
+                    ))}
             </section>
-            <footer></footer>
+            <footer>&copy; take seven 2020</footer>
```

Comment:
Jeeeeez Christ, now I can`t copy anything Ben 😢

Reply:
just wanted to have some text in the footer :-)